settings.gradle

```diff
@@ -1 +1,32 @@
+pluginManagement {
+  repositories {
+    gradlePluginPortal()
+    maven { url = 'https://maven.minecraftforge.net' }
+  }
+
+  resolutionStrategy {
+    eachPlugin {
+      // to support forges old buildscript dsl plugin
+      if (requested.id.namespace == 'net.minecraftforge') {
+        useModule group: 'net.minecraftforge.gradle', name: 'ForgeGradle', version: requested.version
```

Comment:
As I mentioned in another comment `plugins` replaced the old `buildscript` or `apply plugin` DSL. However, not every plugin supports the new DSL (despite it being out for years now). ForgeGradle is one of those plugins. In order to make it work with the new DSL, we must do a little hack. `useModule` is the near equivalent to doing `buildscript.dependencies.classpath`.